### DISCUSSION TITLE
What does this mean?

### DISCUSSION THREAD

**Discussion by JuanAstor95** [zepp-health/zeppos-forum#368] [User Feedback]
Hi everyone, I have a doubt that I cant let it go.
I picked my fav watchface (pic)
![Screenshot_20240909_123938_Zepp](https://github.com/user-attachments/assets/a6725e3f-93b4-4fc9-829e-cbdc88ed1f0e)
What does this leaf mean? Below the umbrella that indicates rain probability 
Where can I find the exact and complete info of every watchface

**Comment by SashaCX75** [comment]
@JuanAstor95
You can only find information about the watch face in its description, which was compiled by the author. There are no other sources of information.
You can look at the data in the weather app and find an indicator with the same value. It is most similar to air quality (AQI).

**Reply by JuanAstor95** [reply]
Author didnt desribed it, it should have nore information for every user that swipes through different watchfaces.
I inmediatly search air quality and the weather app, and yes! Values match and I learn a lot of this indicator, really important cause my place is heading drought and fire all around.
Thank you very much
